src/components/DeleteWarningModal.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+
+
+const DeleteWarningModal = ({ onDelete, onCancel, type }) => (
+  <div className={elementStyles.overlay}>
+    <div className={elementStyles['modal-warning']}>
+      <div className={elementStyles.modalHeader}>
+        <h1>
+          Delete
+          {' '}
+          { type }
+        </h1>
+      </div>
+      <form className={elementStyles.modalContent}>
+        <p>Are you sure you want to delete this?</p>
```

Comment:
This should be styled in accordance with the styles used in other modal sections